src/components/molecules/ZopaFooter/Legal/Legal.tsx

```diff
@@ -13,24 +12,24 @@ const StyledParagraph = styled.p`
 `;
 
 const Legal = () => {
-  const { about, rightsReserved } = {
-    about: `Zopa Limited is incorporated in England & Wales (registration number 05197592), with its registered office at 1st Floor, Cottons Centre, Tooley Street, 
-    London, SE1 2QG. Zopa Limited is authorised and regulated by the Financial Conduct Authority, 
-    and entered on the Financial Services Register under firm registration number 718925.`,
-
-    rightsReserved: `© Zopa Limited ${new Date().getFullYear()} All rights reserved. 'Zopa' and the Zopa logo are trade marks of Zopa Limited. 
-    Zopa is a member of CIFAS – the UK's leading anti-fraud association, 
-    and we are registered with the Office of the Information Commissioner (No. Z8797078).`,
+  const { about, rightsReserved, cifas } = {
+    about: `Zopa Limited is authorised and regulated by the Financial Conduct Authority, and entered on the Financial Services Register (718925). Zopa Bank Limited is authorised by the Prudential Regulation Authority and regulated by the Financial Conduct Authority and the Prudential Regulation Authority, and entered on the Financial Services Register (800542). Zopa Limited (05197592) and Zopa Bank Limited (10627575) are both incorporated in England & Wales and have their registered office at: 1st Floor, Cottons Centre, Tooley Street, London, SE1 2QG.`,
+
+    rightsReserved: `© Zopa Bank Limited ${new Date().getFullYear()} All rights reserved. 'Zopa' is a trademark of Zopa Bank Limited.`,
+
+    cifas: `Zopa is a member of Cifas – the UK’s leading anti-fraud association, and we are registered with the Office of the Information Commissioner (ZA275984, Z8797078).
+
+    `,
   };
 
   return (
     <>
       <Wrapper>
         <StyledParagraph>{about}</StyledParagraph>
-      </Wrapper>
-      <Separator />
-      <Wrapper>
+        <br />
```

Comment:
I think we could avoid `<br />` and instead add some CSS to `<StyledParagraph />` to have _new lines_ 🙃

Reply:
This is just to change the text. After this, a new PR will come with a refactor. That's why we allowed this <br>

Reply:
This has been done, thanks Lluis 👍